test/signup.test.js

```diff
@@ -0,0 +1,127 @@
+const chai = require('chai');
+const app = require('../index');
+const request = require('supertest')(app);
+const db = require('../models/');
+const mocha = require('mocha');
+const faker = require('faker');
+
+const describe = mocha.describe;
+
+const expect = chai.expect; 
+
+
+function generateUser(){
+    const user = {
+        email: faker.internet.email(),
+        username:faker.name.lastName() + Date.now(),
```

Comment:
instead if using `Date.now()` can you just trim out the email part from the `@` symbol and use that for the `username`